enhancedstats-addon.txt

```diff
@@ -212,6 +212,7 @@
 /pubsys.js
 /reporting.do?
 /resonance.js
+/rollbar
```

Comment:
bad filter, slow and too generics.

Reply:
how would you recommend blocking both `rollbar.min.js` and `rollbar.js` as files?

Reply:
```
/rollbar.js
/rollbar.min.js
```
Filters must be very precisely and efficient as possible https://easylist.to/pages/policy.html

Reply:
Thank you! Updated to match your recommendations